vendor/assets/LoginForm1byColorlib/fonts/font-awesome-4.7.0/less/font-awesome.less

```
@@ -0,0 +1,18 @@
+/*!
+ *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
+ *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
+ */
+
+@import "variables.less";
```

c nghĩ là c vừa cmt một chỗ như này rồi chứ nhỉ?
```suggestion
@import 'variables.less';
```